setup.py

```diff
@@ -3,18 +3,23 @@
 
 repo_base_dir = os.path.abspath(os.path.dirname(__file__))
 
+package_about = {}
+with open(os.path.join(repo_base_dir, "tardis", "__about__.py")) as about_file:
+    exec(about_file.read(), package_about)
```

Comment:
Please, avoid using exec.

Reply:
I just committed an adapted version by importing the __about__ file from the explicit path.